tsconfig.json

```diff
@@ -0,0 +1,59 @@
+{
+  "compilerOptions": {
+    "typeRoots": [
+      "./types"
+    ],
+    /* Basic Options */
```

Comment:
the funky formatting is because this file is technically JSON5 format which allows for comments, and GH doesn't render that well